mainwindow.cpp

```diff
@@ -166,7 +167,7 @@ MainWindow::MainWindow(QSettings &settings)
     //     warnAboutDefaultPathToJohn();
     Translator& translator = Translator::getInstance();
     m_ui->comboBox_LanguageSelection->insertItems(0, translator.getListOfAvailableLanguages());
-    m_ui->comboBox_LanguageSelection->setCurrentText(translator.getCurrentLanguage());
+    //TODO:m_ui->comboBox_LanguageSelection->setCurrentText(translator.getCurrentLanguage());
```

Comment:
This one seems to be easy to fix by findText() which return index of the language in the comboBox and setCurrentIndex(int).